.gitignore

```diff
@@ -19,3 +19,9 @@ js/javascript.min.js
 # these are easily recreated on a new project
 # with `npm install`
 node_modules/
+
+# ignore composer dependencies
+# these are easily recreated on a new project
+# with `composer install`
+vendor/
+composer.lock
```

Comment:
If we were loading dependencies (e.g. a PHP library to help our theme), it would make sense to include `composer.lock` so we could lock the version of the library.  Since we're not loading any dependencies we really don't need it.  Nevertheless, I'd like to include `composer.lock` in case.  Arguably it will be good if we ever add a dependency to this theme but really it is just an arbitrary decision.